### PR TITLE
Updated info about Haskell.org, Inc.

### PR DESCRIPTION
**Pull request by mpilgrem**
Given my knowledge as the current Secretary of Haskell.org, Inc., I have suggested some updates to this page for the website Committee to consider, as follows. I have also reformatted existing content to avoid long lines. There are no substantive changes other than those identified in the points below. In particular, I have made no changes to the description of how the Committee itself is constituted or governed.

* For the benefit of non-US readers, I have clarified that '501(c)(3)' is a reference to the US Internal Revenue Code of 1986. I have also distinguished the legal form (New York State Not-for-Profit Corporation) from the US tax status (tax exempt).
* I have used 'the Committee' consistently for the committee (and introduced that term).
* I have used 'the corporation' rather than 'Haskell.org' to distinguish the corporation from the website domain.
* In 'The corporation is formed to ...', I have followed closely the wording as set out in the current bylaws.
* I have provided a link to the current bylaws of the corporation.
* I have included a very brief reference to 'the directors' in addition to the members of the Committee.
* I have corrected the current statement that the members of the Committee all serve as directors of the corporation.
* I have assumed that the Committee does not oversee all donations to the corporation (only 'certain' donations).
* I have reordered the 'history' to be in chronological order.
* As I mention the legal form in the introduction, I do not repeat it in the 'history'.
* I have corrected the date of incorporation, based on the information available at https://apps.dos.ny.gov/publicInquiry/.
* I have added the DOS ID of the corporation.

### CODE REVIEW

**Review by tomjaguarpaw — APPROVED** [review]

**Comment by tomjaguarpaw** [comment]
Since this is a substantial change I think it would be appropriate to get a committee majority before merging. That means we need three more approvals.

**Review by Tritlo — APPROVED** [review]

**Comment by Tritlo** [comment]
Looks good!

**Review by rebeccaskinner — APPROVED** [review]
This looks good to me.

**Review by aaronallen8455 — APPROVED** [review]
This looks good

**Comment by tomjaguarpaw** [comment]
Technically this has now passed review, but I think we should give @angerman, @aviaviavi and @Kleidukos another couple of days to make any comments.

**Comment by Kleidukos** [comment]
Thanks for your patience.

**Review by Kleidukos — APPROVED** [review]

**Comment by mpilgrem** [comment]
Hi! @tomjaguarpaw, just following up on this, given the passage of time and the recent meeting of members of the Committee.

**Comment by tomjaguarpaw** [comment]
Yup, you're right. I didn't mean to leave it this long. Thanks for the reminder!

**Pull request merged**
